src/ducks/splits.js

```diff
@@ -69,7 +70,7 @@ const fetchSplit = (user) => {
       type: FETCH_SPLIT,
     });
 
-    Split.load("wgranger-test/anti-slavery-testing").then((splits) => {
+    Split.load(config.projectSlug).then((splits) => {
```

Comment:
Wait, this should be `config.zooniverseLinks.projectSlug` . Let me post a new update commit.